whiskey.yml

```diff
@@ -34,6 +34,15 @@ Build:
     OnlyDuring: Clean
     Path: reset.ps1
 
+- GetPowerShellModule:
+    Name: Carbon.Core
+    Version: 1.*
+
+- GetPowerShellModule:
+    Name: Carbon.Core
+    Version: 1.0.0
+    Path: Carbon.AccountManagement\Modules

```

Comment:
I don't think we need `Carbon.Core` bundled in this module. I don't see where it's used?

Reply:
I believe we need it in the ConvertTo-SecurityIdentifier tests, used to access the 'Test-COperatingSystem' function.

Reply:
Yeah looks like we need it just for the tests and not for the actual module itself so no need for bundling `Carbon.Core` inside of the `Carbon.AccountManagement`.